dsgrid/dimension/time.py

```diff
@@ -264,18 +327,26 @@ def list_time_range(self):
             list of datetime
 
         """
-        return list(self.iter_timestamps())
+        utc_list = list(self.iter_timestamps())
+        return [ts.astimezone(self.tzinfo) for ts in utc_list]
```

Comment:
Can you do this in 1 loop iteration instead of 2?

Reply:
yes! I missed the yield part of the iter_timestamps(). I was always wondering why this works, since I did not see anything returned.